.github/workflows/winget.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         id: should-publish
 
       - if: ${{ steps.should-publish.outputs.publish == 'true' }}
-        uses: vedantmgoyal2009/winget-releaser@959478928fbc8507d7e5d600c80dfeb3df521537
```

Comment:
@vedantmgoyal2009 what's the history here? According to github, [this commit has been removed from all branches and tags](https://github.com/vedantmgoyal2009/winget-releaser/commit/959478928fbc8507d7e5d600c80dfeb3df521537)

This:
- stops dependabot from tracking the tags
- is not a good practice; removing commits for released versions reduces visibility and trust

Reply:
It looks like you removed this commit intentionally: https://github.com/vedantmgoyal2009/winget-releaser/issues/167#issuecomment-1576968336

**Rewriting history of public branches is a suspicious antipattern** which also breaks many flows - you should generally use `git revert <commit>` instead.